### PR TITLE
Added support for radiator valve Siterwell GS361

### PR DESCRIPTION
**Pull request by dzegarra**
> This radiator valve uses the Tuya platform. 

#### From Zigbee

* Get the target temperature
* Get room temperature
* Read mode (off/antifreeze, heat, and auto)
* Read battery status (0-100)
* Enabled/disabled window detection
* Enabled/disabled valve detection (although I don’t know what this means)
* Enabled/disabled child lock

#### To Zigbee

* Set the target temperature
* Change mode (off/antifreeze, heat, and auto)
* Enable/disable window detection
* Enable/disable valve detection (although I don’t know what this means)
* Enable/disable child lock

### CODE REVIEW

**Comment by Koenkk** [comment]
Thanks!

**Comment by dzegarra** [comment]
Thanks to you. Keep the great work ;)

File: converters/toZigbee.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ const options = {
     },
 };
 
+const siterwellGs361SystemModes = {
```
Comment (sjorge) [inline converters/toZigbee.js:38]: Is this mapping different than the system_mode mapping for other thermostats?

Comment (dzegarra) [reply converters/toZigbee.js:38]: yes. Take a look. The GS361 has only 3 modes and the enumeration is also different.

**Pull request merged**